src/audio/base_fw.c

```diff
@@ -504,6 +504,28 @@ int schedulers_info_get(uint32_t *data_off_size,
 	return 0;
 }
 
+void set_perf_meas_state(const char *data)
+{
+	enum ipc4_perf_measurements_state_set state = *data;
+
+	struct telemetry_wnd_data *wnd_data =
+			(struct telemetry_wnd_data *)ADSP_DW->slots[DW_TELEMETRY_SLOT];
+	struct system_tick_info *systick_info =
+			(struct system_tick_info *)wnd_data->system_tick_info;
+
+	switch (state) {
+	case IPC4_PERF_MEASUREMENTS_DISABLED:
+		break;
+	case IPC4_PERF_MEASUREMENTS_STOPPED:
+		for (int i = 0; i < CONFIG_MAX_CORE_COUNT; i++)
+			systick_info[i].peak_utilization = 0;
```

Comment:
I am not sure what exactly you want for peak_utilization, is it for pipeline or core?
do you have a github feature link for telemetry?